packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -405,7 +405,7 @@ const themeStyleTag = computed(
             v-if="configuration?.isEditable"
             #empty-state>
             <GettingStarted
-              :theme="configuration?.theme || 'default'"
+              :theme="(configuration?.theme as ThemeId) || 'default'"
```

Comment:
```suggestion
              :theme="configuration?.theme || 'default'"
```
we shouldn't need this?

Reply:
also we should rarely use `as` as it bypasses type safety